docker-compose.yml

```diff
@@ -14,6 +14,7 @@ web:
   environment:
     PORT: 8070
     DATABASE_URL: postgres://postgres@db:5432/postgres
+    DEBUG:
```

Comment:
This addition results in DEBUG being False for me in the settings by default. Should this just be True since I don't think we'll be using docker-compose in production?